projects/core/src/lib/util/components/error-page/backend-status-page.component.ts

```diff
@@ -0,0 +1,30 @@
+import {HttpErrorResponse} from '@angular/common/http';
+import {ChangeDetectorRef, Component, OnInit} from '@angular/core';
+import {BackendInfoDict} from '../../../backend/backend.model';
+import {BackendService} from '../../../backend/backend.service';
+
```

Comment:
It would be nice, if we refresh the page, that it would lead back to the map. Because you usually hit refresh to check if it now works.

Reply:
part of [99a0802](https://github.com/geo-engine/geoengine-ui/pull/512/commits/99a080258fa877d66157cf1184739c316b901dc2)